code/ui/blocks/src/blocks/Story.stories.tsx

```diff
@@ -13,8 +14,82 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
-export const BasicOf: Story = {
+export const Of: Story = {
   args: {
     of: BooleanStories.Undefined,
   },
 };
+
+export const OfWithMeta: Story = {
+  args: {
+    of: BooleanStories.True,
+    meta: BooleanStories.default,
+  },
+};
+
+export const Id: Story = {
+  args: {
+    id: `${process.env.BLOCKS_ONLY ? '' : 'storybook-blocks-'}controls-boolean--false`,
+  },
+};
+
+export const Name: Story = {
+  args: {
+    name: 'True',
+  },
+};
+
+export const Inline: Story = {
+  args: {
+    of: BooleanStories.Undefined,
+    inline: true,
+  },
+  decorators: [
```

Comment:
I'm unsure of this border. It means that the story will not be rendered 100 % as the block would so it could cause confusion, however without the border the following stories are indistinguishable from each other unless the user enables the Measure tool.

Reply:
Maybe we'd be better off referencing a story that's purely for display purposes (maybe it renders its size, similar to the measure addon?)

Reply:
Good idea. done.